### PR TITLE
Add a Grunt action to set GP_SCRIPT_DEBUG to false when we deploy to wporg

### LINKED ISSUE #1587: Minification: Improve workflow for Pull Requests that contain changes to JavaScript files
Right now, the minified files easily lead to conflicts when another PR with changes to the JS files has been merged. Can we improve this process?

My suggestion:

Remove minified files from develop and only generate and use them upon deploy to WordPress.org. This means we'd delete the minified files from the develop branch, change the `SCRIPT_DEBUG` if condition to add another constant and set this constant before deploying to .org.

e.g. `$suffix = SCRIPT_DEBUG || ! GP_USE_MINIFIED ? '.css' : '.min.css';`

### PR DESCRIPTION
**Pull request by amieiro**
## Problem

When we deploy a new version to wporg, we need to minify the CSS and the JavaScript tiles. 

- In https://github.com/GlotPress/GlotPress/pull/1614 we added a `GP_SCRIPT_DEBUG` constant to load the unminified CSS and JavaScript files by default, setting this `GP_SCRIPT_DEBUG` constant to `true` in the **glotpress.php** file.
- In https://github.com/GlotPress/GlotPress/pull/1654 and in https://github.com/GlotPress/GlotPress/pull/1655 we added some actions to minify this files when we deploy to wporg.

But we need to change the `GP_SCRIPT_DEBUG` constant to `false` before we deploy to wporg, so the wporg release will use the minified files (except the end user will use the `SCRIPT_DEBUG` set to `true`). This PR address this problem.

Fixes https://github.com/GlotPress/GlotPress/issues/1587.

## Solution

This PR adds:
- A **Grunt action** to change the `define( 'GP_SCRIPT_DEBUG', true );` to `define( 'GP_SCRIPT_DEBUG', false );` in the glotpress.php file. **Gruntfile.js** file.
- A **npm action** to run the Grunt action. **package.json** file.
- This npm action into the **wporg workflow**. **.github/workflows/deploy-wporg-plugin.yml** file.

## Testing Instructions

To test this PR:
- You can execute the Grunt action with the `grunt replace:updateGPScriptDebugConst` command.
- You can execute the npm action with the `npm run updateGPScriptDebugConst` command.

In both situations, you can check that the `GP_SCRIPT_DEBUG` value in the `glotpress.php` file has changed from true to false.

### CODE REVIEW

File: .github/workflows/deploy-wporg-plugin.yml [hunk 1]
```diff
@@ -28,6 +28,7 @@ jobs:
         run: |
           npm ci
           npm run build
+          npm run updateGPScriptDebugConst
```
Comment (akirk) [inline .github/workflows/deploy-wporg-plugin.yml]:
```suggestion
          npm run prepare-release
```

File: Gruntfile.js [hunk 2]
```diff
@@ -67,6 +67,18 @@ module.exports = function( grunt ) {
 				tasks: [ 'cssmin' ],
 			},
 		},
+		replace: {
+			updateGPScriptDebugConst: {
```
Comment (akirk) [inline Gruntfile.js]:
```suggestion
			prepare-release: {
```

File: package.json [hunk 3]
```diff
@@ -19,11 +19,13 @@
     "grunt-contrib-cssmin": "^5.0.0",
     "grunt-contrib-uglify": "^5.2.2",
     "grunt-contrib-watch": "^1.1.0",
+    "grunt-text-replace": "^0.4.0",
     "load-grunt-tasks": "^5.1.0"
   },
   "scripts": {
     "build": "grunt",
     "watch": "grunt watch",
+    "updateGPScriptDebugConst": "grunt replace:updateGPScriptDebugConst",
```
Comment (akirk) [inline package.json]:
```suggestion
    "prepare-release": "grunt replace:prepare-release",
```

**Review by akirk — COMMENTED** [review]
What about this?

**Comment by amieiro** [comment]
> What about this?

With the dash in the Grunt action name, I have problems running the task in my local machine, so I have added single quotation marks to this action name.

```
npm run prepare-release

> glotpress@0.0.0 prepare-release
> grunt replace:prepare-release

Loading "Gruntfile.js" tasks...ERROR
>> SyntaxError: Unexpected token '-'
Warning: Task "replace:prepare-release" not found. Use --force to continue.

Aborted due to warnings.
```

**Review by akirk — APPROVED** [review]
Thank you!

**Pull request merged**